app/src/main/java/ly/loud/loudly/application/models/GetterModel.java

```diff
@@ -44,25 +58,89 @@ public GetterModel(
         this.coreModel = coreModel;
     }
 
+    @NonNull
+    private PublishSubject<Pair<SingleNetworkElement, Throwable>> getGetErrors() {
+        if (getErrors == null) {
+            getErrors = PublishSubject.create();
+        }
+        return getErrors;
+    }
+
+    @CheckResult
+    @NonNull
+    public Observable<Pair<SingleNetworkElement, Throwable>> observeGetErrors() {
+        return getGetErrors().asObservable();
+    }
+
+    @NonNull
+    private PublishSubject<Pair<SingleNetworkElement, Throwable>> getGetCommentsErrors() {
+        if (getCommentsErrors == null) {
+            getCommentsErrors = PublishSubject.create();
+        }
+        return getCommentsErrors;
+    }
+
+    @CheckResult
+    @NonNull
+    public Observable<Pair<SingleNetworkElement, Throwable>> observeGetCommentsErrors() {
+        return getGetCommentsErrors().asObservable();
+    }
+
+    @CheckResult
+    @NonNull
+    private Observable<PersonsFromNetwork> safeGetPersonsByType(@NonNull SingleNetworkElement element,
+                                                                @RequestType int type,
+                                                                @NonNull NetworkContract networkContract) {
+        return retry3TimesAndFail(
+                networkContract
+                        .getPersons(element, type)
+                        .map(persons -> new PersonsFromNetwork(persons, networkContract.getId())),
+                new FatalNetworkException(networkContract.getId()))
+                .doOnError(error -> getGetErrors().onNext(new Pair<>(element, error)))
+                .onErrorResumeNext(Observable.empty());
+    }
+
+    @CheckResult
+    @NonNull
+    private Observable<CommentsFromNetwork> safeGetComments(@NonNull SingleNetworkElement element,
+                                                            @NonNull NetworkContract networkContract) {
+        return retry3TimesAndFail(
+                networkContract
+                        .getComments(element)
+                        .map(comments -> new CommentsFromNetwork(comments, networkContract.getId())),
+                new FatalNetworkException(networkContract.getId()))
+                .doOnError(error -> getGetCommentsErrors().onNext(new Pair<>(element, error)))
+                .onErrorResumeNext(Observable.empty());
```

Comment:
Suppose we can't get comments from 1 network, but we are able to get it from other networks. If we propagate error futher, it will fail other networks